packages/controller-utils/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [6.2.0]
 ### Changed
```

Comment:
For some reason the "safely execute type safety" change was missed here. I'll add that now, but I am not sure how it got missed...

Reply:
Added in https://github.com/MetaMask/core/pull/3667/commits/abf8009c6d8668675a6264dcd071709741ed4e02